src/loader/ComponentLoader.stories.js

```diff
@@ -1,18 +1,12 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
-import { checkA11y } from '@storybook/addon-a11y'
-// import { action } from '@storybook/addon-actions'
-import { withKnobs, boolean } from '@storybook/addon-knobs'
-
 import ComponentLoader from './ComponentLoader'
 
-storiesOf('Loader', module)
-  .addDecorator(checkA11y)
-  .addDecorator(withKnobs)
-  .add('ComponentLoader', () => (
-    <div className='sans-serif pa4' style={{ height: 400 }}>
-      <ComponentLoader
-        pastDelay={boolean('pastDelay', true)}
-      />
+storiesOf('ComponentLoader', module)
+  .add('Default', () => (
+    <div className='flex items-center'>
+      <ComponentLoader pastDelay style={{ width: 100, margin: '100px auto' }}/>
+      <ComponentLoader pastDelay style={{ width: 200, margin: '100px auto' }}/>
+      <ComponentLoader pastDelay style={{ width: 300, margin: '100px auto' }}/>
```

Comment:
Why we have three of those?

Reply:
I think this was just to get a sense of the cube's visual impact at a few different sizes, as it may appear at different scale in different settings. Could remove, but it was useful to see in our implementation discussion.

Reply:
ah, I was before my coffe and did not realize this is storybook :sweat:  nvm me!